src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/RestrictedPatientsSearchResourceTest.kt

```diff
@@ -291,6 +291,14 @@ class RestrictedPatientsSearchResourceTest : QueueIntegrationTest() {
         "/results/restrictedPatientsSearch/search_results_supporting_prison_DNI_MDI.json"
       )
     }
+
+    @Test
```

Comment:
Used this ticket to add another restricted patient test to complete the possible combinations of searches.